suricata.yaml.in

```diff
@@ -449,6 +455,9 @@ outputs:
       # remain open for filestore by Suricata. Default value is 0 which
       # means files get closed after each write
       #max-open-files: 1000
+      # uncomment to write to tmp files with the given prefix
+      #working-file-prefix: .
+      include-pid: no # set to yes to include pid in file names
```

Comment:
Can you add working-file-prefix and include-pid to the user guide section on file extraction? I think these are important features - I think they should be the default, but that is unlikely at the moment for compatibility reasons.